queries.sql

```diff
@@ -1,3 +1,17 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM public.animals WHERE name LIKE '%mon';
+
+SELECT name FROM public.animals WHERE date_of_birth BETWEEN '01-01-2016' AND '12-31-2019';
+
+SELECT name FROM animals WHERE neutered = TRUE AND escape_attempts < 3;
+
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
+
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+
+SELECT * FROM animals WHERE neutered;
```

Comment:
**Changes Required**
- [ ] This should return animals where `neutered` is true. Currently it just returns nothing. Let us try and fix this bug by ensuring the `WHERE CLAUSE` says `true`..